test/type_test.js

```diff
@@ -1,67 +1,150 @@
-/**
- * Copyright 2012 Google Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-/**
- * @fileoverview Unit tests for type().
- * @author bnkuhn@gmail.com (Brian Kuhn)
- */
-
-test("type", function() {
+goog.require('plain');
+
+describe('The plain.type function', function() {
 
   function assertType(value, expected) {
-    equal(plain.type(value), expected);
+    expect(plain.type(value)).toBe(expected);
```

Comment:
It seems like every expect statement in the file is wrapped with it('..'). It becomes hard to read compared to the old approach.

Could you consider replacing describe() with it() and optionally pass context message in the assertion instead using withContext() something like

```
function assertType(value, expected, context = '') {
    expect(plain.type(value)).withContext(context).toBe(expected);
}

it('correctly identifies null and undefined', function() {
      assertType(null, 'null', 'identifies null');
      assertType(undefined, 'undefined', 'identifies undefined');
});

it('correctly identifies boolean', function() {
    assertType(true, 'boolean');	      
    assertType(false, 'boolean');
});
```

Reply:
Ok, I wasn't sure since I had heard that all asserts should be separate test cases before but this makes sense, I'll go back through and change them